client/src/pages/Lobby/Lobby.tsx

```diff
@@ -2,21 +2,21 @@ import React, { FC, useEffect, useState } from 'react'
 import axios, { AxiosResponse } from 'axios'
 import moment from 'moment'
 import { RouteComponentProps } from '@reach/router'
-
+import { Settings as SettingsType } from '../../types/Settings'
 import connectToGameHub from '../../utils/signalrConnector'
 import { Game } from '../../types/Game'
-import { Settings as SettingsType } from '../../types/Settings'
-import { Difficulty } from '../../types/Difficulty'
 import Settings from '../../containers/Settings'
 import './Lobby.scss'
 import PlayerBox from '../../components/PlayerBox'
+import { HubConnection } from '@microsoft/signalr'
```

Comment:
Split local and external imports:

externals
*< line break >*
locals

Reply:
I blame auto imports 😄